python/jupytergis_lab/jupytergis_lab/notebook/gis_document.py

```diff
@@ -318,6 +318,102 @@ def add_video_layer(
 
         self._add_layer(OBJECT_FACTORY.create_layer(layer, self))
```

Comment:
In terms of usability, it seems the user has no easy means to get the layer id. So we should probably return the id for all methods that create a layer. That way the user can retrieve the id and apply filters right away. 

```suggestion
        return self._add_layer(OBJECT_FACTORY.create_layer(layer, self))
```

Reply:
Also, I'm thinking it may be nice to provide filters directly as a parameter of `def add_geojson_layer` or `def add_vectortile_layer(`